README.md

```diff
@@ -9,7 +9,22 @@ You can see details of [the project roadmap here](https://napari.org/roadmaps/in
 
 ## contributing
 
-Contributions are encouraged! Please read our [contributing guide](https://napari.org/dev/developers/contributing/documentation/index.html) to get started. Given that we're in an early stage, you may want to reach out on our [Github Issues](https://github.com/napari/docs/issues) before jumping in.
+Contributions are encouraged! Given that we're in an early stage, you may want to reach out on our [Github Issues](https://github.com/napari/docs/issues) before jumping in.
+It's nice to be able preview how your contribution will look on the web, so here we provide a brief summary of what's required for a local setup that will allow you to do so.
+Please read our [contributing guide](https://napari.org/dev/developers/contributing/documentation/index.html) for more comprehensive information about contributing documentation.
+
+### quickstart: local setup
+* __Check the prerequisites__
+    1. Create a [development installation](https://napari.org/stable/developers/contributing.html#dev-installation) of `napari` on your local machine.
+    1. Fork this repository, and then clone your fork to your local machine. 
+NB: the name will be `napari-docs` rather than simply `docs`.
```

Comment:
```suggestion
NB: you may want to name your fork e.g. `napari-docs` rather than just `docs`.
```
Pretty sure *by default* it's called `docs` when you fork, but renaming it is a good idea IMO.

Reply:
Ah yeah I think I mixed up what happens when it's cloned vs. when it's forked, but I like this suggestion, I think it's nice to have the fork named that way anyhow.